test/functionalTest/cases/0501_cors/allowed_methods.test

```diff
@@ -109,168 +109,169 @@ echo
 01. Entry Points - Allowed methods: GET OPTIONS
```

Comment:
Please review the modifications in this test. It seems something is failing https://github.com/telefonicaid/fiware-orion/actions/runs/4919583948/jobs/8876244422

```
 ----------- Failing tests ------------------
  o  0000_subs_attrs_deletion/delete_attribute_not_in_condition.test
  o  0501_cors/allowed_methods.test
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4321/commits/c7478f401c0466246950316809c9ec5521bb1ee1